src/modules/85-iac/components/IacResourceStackWizard/StackProvisionerTypeStep.tsx

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2022 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+import React, { useMemo } from 'react'
+import { Button, ButtonVariation, Formik, FormikForm, IconName, Layout, Text, ThumbnailSelect } from '@harness/uicore'
+import { FontVariation } from '@harness/design-system'
+import * as Yup from 'yup'
+import { useStrings } from 'framework/strings'
+import { NameIdDescriptionTags } from '@common/components'
+import type { IacResourceStack, StackWizardStepProps } from '.'
+import css from './StackWizard.module.scss'
+
+type Provisioner = {
+  name: string
+  icon: IconName
+  value: string
+}
```

Comment:
Just preference, but this looks more suited to an interface than a type
```suggestion
interface Provisioner {
  name: string
  icon: IconName
  value: string
}
```

Reply:
Updated to interface